src/api/Reducers/deltaTimes.js

```diff
@@ -0,0 +1,36 @@
+import { actionTypes } from '../Actions/actionTypes';
+
+const defaultState = {
+  hasNextDeltaTimeStep: undefined,
+  hasPrevDeltaTimeStep: undefined,
+  nextDeltaTimeStep: undefined,
+  prevDeltaTimeStep: undefined
+};
+
+// TODO: this is where I should change how the data I send from the topic should be converted to states in the store
```

Comment:
Maybe specify how and why this should be changed if still applicable?
To me it seems a bit unclear what should be done.

Reply:
Just had this comment as a reminder to myself during development and forgot to remove it. Woops ^^